### PR TITLE
[CI] Add LLAMA3 ACC CI Test

### PR DESCRIPTION
**Pull request by intellinjun**
## Type of Change

feature or bug fix or documentation or others
API changed or not
not
## Description
Add llama2 model acc ut for piqa task, llama2 rtn quant config is wint4 cint8 group_size 32

### CODE REVIEW

File: tests/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+gguf
```
Comment (zhentaoyu) [inline tests/requirements.txt:1]: it has already been added in setup.py. please remove it.

**Comment by zhentaoyu** [comment]
@VincyZhang, could you please have a look at this? Thanks.

File: .github/workflows/scripts/models/cpp_graph_inference.sh [hunk 2]
```diff
@@ -139,6 +145,7 @@ function main() {
                     NEURAL_SPEED_VERBOSE=1 OMP_NUM_THREADS=$(($cores_per_instance * 1)) numactl -m 0 -C 0-$(($cores_per_instance * 1 - 1)) \
                         $infer_cmd --seed 1234 -t $cores_per_instance -b 2047 -c ${ctx} -n ${output} -m ${model}-${precision}.bin -p "$prompt" 2>&1 | tee ${WORKING_DIR}/${logs_file} || true & 
                     monitor
+                    python ./scripts/cal_acc.py --model_name ${input_model} --init_from_bin ${model}-${precision}.bin --tasks lambada_openai --batch_size 8  2>&1 | tee ${WORKING_DIR}/${logs_file}
```
Comment (zhentaoyu) [inline .github/workflows/scripts/models/cpp_graph_inference.sh:148]: only ”llama" and "gpt-j" support batch_size > 1. We may need a filter.

Comment (intellinjun) [reply .github/workflows/scripts/models/cpp_graph_inference.sh:148]: we only use llama and gpt-j in ci

**Review by a32543254 — APPROVED** [review]
LGTM

**Review by zhentaoyu — APPROVED** [review]
Thanks for your effort.  LGTM.

**Comment by intellinjun** [comment]
https://inteltf-jenk.sh.intel.com/job/neural_speed_extension/159/artifact/report.html
acc extension test result

**Pull request merged**